content/platforms/kubernetes/pod-scheduling.md

```diff
@@ -224,6 +204,7 @@ spec:
   rackAwarenessNodeLabel: failure-domain.beta.kubernetes.io/zone
 ```
 
-The operator will change the `topologyKey` for the pod anti-affinity rule to
-the label value used. Combining `rackAwarenessNodeLabel` and `podAntiAffinity`
-requires care to ensure you make the same change.
+{{% note %}}
+When you use the `rackAwarenessNodeLabel`, the operator changes the `topologyKey` for the pod anti-affinity rule to the label value used.
+If you use `rackAwarenessNodeLabel` and `podAntiAffinity` together, make sure that the resulting `topologyKey` is as expected.
+{{% /note %}}
```

Comment:
No. It does not change the topologyKey if you have changed the anti-affinity rules as well. You have to do that yourself.

"When you use the `rackAwarenessNodeLabel`, the operator will change the `topologyKey` for the anti-affinity rule to the label value used unless you have specified the `podAntiAffinity` as well. If you use `rackAwarenessNodeLabel` and `podAntiAffinity` tougher, you must make sure that the `topologyKey` in your pod anti-affinity rule is set to the node label."